examples/gsuite-mcp/google-doc-mcp-example.py

```
@@ -0,0 +1,60 @@
+from openai import OpenAI
+from google_auth_oauthlib.flow import InstalledAppFlow
+
+
+PROMPT = "Using the tool provided, create a document containing a poem and return the link to it"
+
+
+def main():
+    client_id = os.getenv("GOOGLE_OAUTH_CLIENT_ID")
+    client_secret = os.getenv("GOOGLE_OAUTH_CLIENT_SECRET")
+    if not any([client_id, client_secret]):
+        raise ValueError("GOOGLE_OAUTH_CLIENT_ID and GOOGLE_OAUTH_CLIENT_SECRET must be set in env")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)